src/Core/Shop.Core.Business/Interfaces/IProductService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Shop.Core.DomainEntities;
+
+namespace Shop.Core.Business.Interfaces;
+
+    public interface IProductService
```

Comment:
all the void methods should return true or false to indicate the success of the call.